pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/transformer/MetsData.scala

```diff
@@ -110,26 +110,49 @@ case class MetsData(
 
   private def thumbnail(maybeLicense: Option[License], bnumber: String) =
     for {
-      location <- thumbnailLocation
-      url <- buildThumbnailUrl(location, bnumber)
+      fileObject <- fileObjects.headOption
+      url <- buildImageUrl(bnumber, fileObject)
     } yield
       DigitalLocation(
         url = url,
         locationType = LocationType("thumbnail-image"),
         license = maybeLicense
       )
 
-  private def buildThumbnailUrl(location: String, bnumber: String) = {
-    val mediaType =
-      URLConnection.guessContentTypeFromName(new File(location).getName)
-    Option(mediaType) match {
+  private val images = fileObjects
+    .filter {
+      _.mimeType match {
+        case Some(m) => m == "application/pdf" || m.startsWith("image")
+        case None    => true
+      }
+    }
+    .map { fileObject =>
+      UnmergedImage(
+        getImageSourceId(recordIdentifier, fileObject.id),
+        DigitalLocation(
+          url = buildImageUrl(recordIdentifier, fileObject).getOrElse(""),
```

Comment:
I think image urls need to be slightly different. This builds the url to the thumbnail which has 200X200 fixed size. I think it should be something like `https://dlcs.io/iiif-img/wellcome/${space}/${filename}/info.json` so you have access to the full size image or you can request whichever size you want

Reply:
What do you think we should do for PDFs? We could use the thumbnails, or just return the whole PDF (which seems to be in DLCS for examples I've looked at, is this universal?). If the latter, then I'm not sure what `LocationType` the `DigitalLocation` should have - it doesn't seem like it should be `iiif-image`.